d2app/app.go

```diff
@@ -829,7 +829,7 @@ func (a *App) quitGame() {
 }
 
 func (a *App) enterGuiPlayground() {
-	a.screen.SetNextScreen(d2gamescreen.CreateGuiTestMain(a.renderer, a.guiManager, a.asset))
+	a.screen.SetNextScreen(d2gamescreen.CreateGuiTestMain(a.renderer, a.guiManager, d2util.LogLevelDefault, a.asset))
 }
 
```

Comment:
we should use the log level that is defined in the config

Reply:
ok @gravestanch I've corrected it.